forge/src/executor/inspector/cheatcodes/expect.rs

```diff
@@ -0,0 +1,203 @@
+use super::Cheatcodes;
+use crate::abi::HEVMCalls;
+use bytes::Bytes;
+use ethers::{
+    abi::{AbiEncode, RawLog},
+    types::{Address, H256},
+};
+use once_cell::sync::Lazy;
+use revm::{return_ok, Database, EVMData, Interpreter, Return};
+use std::str::FromStr;
+
+/// For some cheatcodes we may internally change the status of the call, i.e. in `expectRevert`.
+/// Solidity will see a successful call and attempt to decode the return data. Therefore, we need
+/// to populate the return with dummy bytes so the decode doesn't fail.
```

Comment:
could  also add comment that 320 bytes was arbitrarily chosen because it's "long enough" for 10 words in the return value of the call being mocked?